BUILD.md

```diff
@@ -170,32 +170,43 @@ Until then, we advise Windows developers to use Visual Studio 2019.
 
 ## Prerequisites
 
-To build this package, you will need Python (>= 3.7),
-[Conan][] (>= 1.55), and [CMake][] (>= 3.16).
-
 > **Warning**
-> The commands in this document are not meant to be blindly copied and pasted.
+> The commands in this document are guidelines which cannot always be successfully copy and pasted.
 > This document is written for multiple audiences,
 > meaning that your particular circumstances may require some commands and not
 > others.
-> You should never run any commands without understanding what they do
-> and why you are running them.
+> It is recommended to fully understand what the following commands do before running them.
 >
 > These instructions assume a basic familiarity with Conan and CMake.
 > If you are unfamiliar with Conan,
 > then please read the [crash course](#a-crash-course-in-cmake-and-conan)
 > at the beginning of this document,
 > or the official [Getting Started][3] walkthrough.
 
+### All Platforms
+
+To build this package, you will need [Python][] (>= 3.7),
+[Conan][] (>= 1.52), and [CMake][] (>= 3.16).
+If you are unfamiliar with Conan,
+there is a crash course at the end of this document.
+
 [Conan]: https://conan.io/downloads.html
 [CMake]: https://cmake.org/download/
+[Python]: https://www.python.org/downloads/
+
+If you've never used Conan before, use autodetect to setup a default profile:
```

Comment:
```suggestion
If you've never used Conan before, you can create a new profile, detecting the settings:
```
phrase copied from https://docs.conan.io/en/1.43/reference/commands/misc/profile.html

Also, no need to say anything about this, but as a heads up, I found this via google:

> I would also like to highlight that for production the profile auto-detection should not be trusted. Using your own profiles (probably managed with conan config install) is the recommended approach. To better emphasize this behavior, Conan 2.0 is also removing the automatic detection and creation of a default profile when it doesn't exist, and it will require a explicit command in 2.0 to detect and create a profile, that will include warnings that such profile output will not be considered stable, and subject to change its defaults from release to release)

~ https://github.com/conan-io/conan/issues/10185#issuecomment-996270361